src/components/ui/BlockQuote/stories/BlockQuote.stories.tsx

```
@@ -39,3 +42,38 @@ export const Color = {
+    return <SandboxEditor>
+        <div className='mt-4 mb-2'>
+            <p className='text-gray-950'>BlockQuote Variant</p>
+        </div>
+        <div className='flex flex-col gap-3'>
+            {/* TODO */}
+            <BlockQuote>
+                <div>{BLOCKQUOTE_TEXT}</div>
+            </BlockQuote>
+       </div>
+    </SandboxEditor>;
+};
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)